src/gobprepare/prepare_client.py

```diff
@@ -248,6 +248,11 @@ def action_check_row_counts(self, action: RowCountConfig) -> bool:
                     f"Deviation of {deviation:.2f}% for {table}! Expected {expected_count} rows, got {row_count}"
                 )
                 tables_counts_ok = False
+            else:
+                logger.info(
+                    f"Row count for table {table} succeeded:"
+                    f" expected {expected_count} rows, got {row_count} ({deviation:.2f}%)"
```

Comment:
f" expected {expected_count} rows, got {row_count} with deviation of ({deviation:.2f}%)"

Reply:
Aangepast